client/jetpack-cloud/sections/settings/advanced-credentials/credentials-form/index.tsx

```diff
@@ -86,14 +90,13 @@ const ServerCredentialsForm: FunctionComponent< Props > = ( { onCredentialsSave,
 
 	const renderPrivateKeyForm = () => (
 		<FormFieldset className="credentials-form__kpri">
-			{ /* <FormLabel htmlFor="private-key">{ labels.kpri || translate( 'Private Key' ) }</FormLabel> */ }
-			<FormLabel htmlFor="private-key">{ translate( 'Private Key' ) }</FormLabel>
+			<FormLabel htmlFor="private-key">{ translate( 'Private key' ) }</FormLabel>
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Show Private Key' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/new-credentials-route)